### PR TITLE
updated contributor.txt

### PR DESCRIPTION
**Pull request by swarupn17**
Issue: #1 
Description: Added Name & Github ID in contributor.txt file

### CODE REVIEW

File: package-lock.json
Comment (aashray374) [inline package-lock.json:1]: Why are you creating this file?

Comment (swarupn17) [reply package-lock.json:1]: Sorry it was by mistake.
should i delete it by make new branch or make a new PR ?

Comment (aashray374) [reply package-lock.json:1]: yes

Comment (aashray374) [reply package-lock.json:1]: delete ti and create a new PR

**Comment by Athting** [comment]
Issue: #1

File: package-lock.json
Comment (swarupn17) [reply package-lock.json:1]: created a new PR please check

**Pull request closed without merging**